test-harness/specs/security-scheme-validation-AC-6_failing.oas2.txt

```diff
@@ -0,0 +1,34 @@
+====test====
+Making a request (and not including valid credentials) to an endpoint,
```

Comment:
Assumed: since 403 examples ended up in the reviewed ACs, I came up with this. But not sure if should do so (?).
How would we know that dGVzdDp0ZXN0 is valid?